rpcserver.go

```diff
@@ -5047,3 +5055,110 @@ func (r *rpcServer) SubscribeChannelBackups(req *lnrpc.ChannelBackupSubscription
 		}
 	}
 }
+
+// ChannelAcceptor dispatches a bi-directional streaming RPC in which
+// OpenChannel requests are sent to the client and the client responds with
+// a boolean that tells LND whether or not to accept the channel. This allows
+// node operators to specify their own criteria for accepting inbound channels
+// through a single persistent connection. There may only be one client at a
+// time for this RPC.
+func (r *rpcServer) ChannelAcceptor(stream lnrpc.Lightning_ChannelAcceptorServer) error {
+
+	var acceptorClient *channelacceptor.ChanAcceptorClient
+	var rpcAcceptor *channelacceptor.RPCAcceptor
+
+	// If the OpenChannelPredicate is the ChainedAcceptor, locate the RPCAcceptor
+	// and register this RPC client.
+	openChanPredicate := r.server.fundingMgr.cfg.OpenChannelPredicate
+	chainedAcceptor, ok := openChanPredicate.(*channelacceptor.ChainedAcceptor)
+	if !ok {
+		// The funding manager is not using the ChainedAcceptor.
```

Comment:
We'll _always_ have the chained acceptor active if this RPC method is also activated?

Reply:
Since the funding manager accepts a `ChannelAcceptor`, I thought that there could be a case where other implementations besides the `ChainedAcceptor` could also be passed in.  But it seems to me with the current design, `ChainedAcceptor` should be the only implementation that should be passed in since it allows more flexibility.  I can change this and the fundingmanager's `start` and `stop` methods.